docs/3.x/tech/installation-guide/cleanspeak-webservice.adoc

```diff
@@ -205,7 +205,7 @@ Find the license Id field at the top of this configuration file and enter your l
 license.id=
 ----
 
-Find the default database JDBC URL, username, and password values, and verify this information is correct. The default JDBC URL is configured for MySQL. If you’re using PostgreSQL, you’ll need to update the URL. See the database.URL property documentation in Configuration Reference for more information.
+Find the default database JDBC URL, username, and password values, and verify this information is correct. The default JDBC URL is configured for MySQL. If you’re using PostgreSQL, you’ll need to update the URL. See the `database.URL` property documentation in link:../reference/configuration[Configuration] for more information.
```

Comment:
In the above two examples you used "Configuration Reference" should we keep them consistent?